src/components/UI/textField/TextField.tsx

```diff
@@ -42,13 +39,15 @@ function TextField(props: ITextFieldProps) {
           render={({ field }) => (
             <TextFieldM
               {...field}
+              disabled={disabled}
               error={!!errors?.[`${formFieldName}`]?.message}
               fullWidth
+              sx={helperText ? undefined : sxProps}
```

Comment:
Despite the fact that it works, typescript gives an error. Because `sx={helperText ?? sxProps}` posibly return string, which is not allowed here.
Left unchanged